# direction -1 is inward, +1 is outward, 0 is stationary
string task_type
time created_at
int8 table_number
int8 customerID
bool finished